consensus/metrics.go

```diff
@@ -0,0 +1,24 @@
+package consensus
+
+import (
+	"github.com/go-kit/kit/metrics"
+	prmkit "github.com/go-kit/kit/metrics/prometheus"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	Namespace = "spacemesh"
+	Subsystem = "consensus"
+)
+
+func getGauge(name, help string, labels []string) metrics.Gauge {
+	return prmkit.NewGaugeFrom(prometheus.GaugeOpts{Namespace: Namespace, Subsystem: Subsystem, Name: name, Help: help}, labels)
+}
+
+var (
+	pbaseCount     = getGauge("pbase_counter", "pbase index", []string{})
```

Comment:
because this is an incremental value it should probably be a counter

Reply:
I've looked at the way you use it and it might make sense to leave it as a gauge. since you don't inc one at a time